info.puredata.Pd-extended.yaml

```diff
@@ -0,0 +1,117 @@
+app-id: info.puredata.Pd-extended
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: pd-extended
+finish-args:
+# Display
+  - --share=ipc
+  - --socket=x11
+  - --device=dri
+# Audio
+  - --socket=pulseaudio
+  - --share=network
+  - --filesystem=xdg-run/pipewire-0
+# Filesystem access (pd doesn't yet work with portals)
+  - --filesystem=host
+  - --talk-name=org.freedesktop.Notifications
+  - --talk-name=org.freedesktop.ScreenSaver
+# MIDI access (Flatpak doesn't yet support exposing only MIDI devices)
+  - --device=all
+# Linux Audio plugins
+  - --env=LADSPA_PATH=/app/extensions/Plugins/ladspa:/app/lib/ladspa
+add-extensions:
+  org.freedesktop.LinuxAudio.Plugins:
+    directory: extensions/Plugins
+    version: '21.08'
+    add-ld-path: lib
+    merge-dirs: ladspa
+    subdirectories: true
+    no-autodownload: true
+
+modules:
+  - shared-modules/lua5.1/lua-5.1.5.json
+#  - shared-modules/SDL/SDL-1.2.15.json # for pdp_sdl
+#  - shared-modules/SDL/SDL_image-1.2.12.json
```

Comment:
Please just remove it if it's not needed.

Reply:
It is needed for some extensions "pdp_sdl" that I hope to also include.